### PR TITLE
PaymentRequest "paymentmethodchanged" event

### PR DESCRIPTION
**Pull request by marcoscaceres**
Tests for https://github.com/w3c/payment-request/pull/695

### CODE REVIEW

File: payment-request/onpaymentmenthodchange-attribute.https.html
Comment (mnoorenberghe) [inline payment-request/onpaymentmenthodchange-attribute.https.html]: I've noticed you remove this comment before and it's not included on the other two tests.

File: payment-request/onpaymentmenthodchange-attribute.https.html
Comment (mnoorenberghe) [inline payment-request/onpaymentmenthodchange-attribute.https.html]: Nit: I thought generally the assertion message should say what should be true (e.g. "must be same event") but for many you are indicating what doesn't happen if it fails. 

e.g. this should be "event fired"

**Review by mnoorenberghe — APPROVED** [review]

File: payment-request/onpaymentmenthodchange-attribute.https.html
Comment (marcoscaceres) [reply payment-request/onpaymentmenthodchange-attribute.https.html]: Yeah, it's a bit all over the place and a matter of preference. The messages only show up when tests fail, so I tend to explain why something failed rather than why something passed. So:

Why did X fail? ..."event did not fire".

**Comment by marcoscaceres** [comment]
Blocking merge, waiting on ok from implementers on the spec side.

**Comment by marcoscaceres** [comment]
updated tests to match updated proposal.

File: lint.whitelist
Comment (gsnedders) [inline lint.whitelist]: This seems wrong. Why is it getting picked up as a visual test?

File: payment-request/PaymentMethodChangeEvent/methodName-attribute.https.html
Comment (gsnedders) [inline payment-request/PaymentMethodChangeEvent/methodName-attribute.https.html]: This needs to be `<script src=`

**Review by gsnedders — DISMISSED** [review]

File: payment-request/PaymentMethodChangeEvent/methodName-attribute.https.html
Comment (marcoscaceres) [reply payment-request/PaymentMethodChangeEvent/methodName-attribute.https.html]: oh, woopsy...

File: lint.whitelist
Comment (marcoscaceres) [reply lint.whitelist]: agree... filed #11479 because this is not documented.

Comment (marcoscaceres) [reply lint.whitelist]: (reverting this, btw)

**Comment by marcoscaceres** [comment]
@gsnedders, take another look.

**Pull request merged**
